content-security-policy/embedded-enforcement/subsumption_algorithm-strict_dynamic.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Subsumption Algorithm - 'strict-dynamic' keyword.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "'strict-dynamic' is ineffective for `style-src`.", 
+        "required_csp": "style-src http://example1.com/foo/ 'self'", 
+        "returned_csp_1": "style-src 'strict-dynamic' http://example1.com/foo/bar.html",
+        "returned_csp_2": null,
```

Comment:
Nit: Just drop these. Without an explicit definition, `test.returned_csp_2` will be `undefined`, which is falsey.